### PR TITLE
Allow using aliased attributes with `insert_all`/`upsert_all`

### PR DESCRIPTION
**Pull request by fatkodima**
```ruby
class Book < ApplicationRecord
  alias_attribute :title, :name
end

Book.insert_all [{ title: "Remote", author_id: 1 }], returning: :title
```

Small convenience. Currently, it raises `unknown attribute 'title' for Book. (ActiveModel::UnknownAttributeError)`

### CODE REVIEW

File: activerecord/lib/active_record/insert_all.rb [hunk 1]
```diff
@@ -88,6 +90,19 @@ def keys_including_timestamps
     private
       attr_reader :scope_attributes
 
+      def resolve_attribute_aliases
```
Comment (byroot) [inline activerecord/lib/active_record/insert_all.rb:93]: Do you think we could have a fast path where we check if the first record contain any aliases, and if not we just skip that method?

Comment (fatkodima) [reply activerecord/lib/active_record/insert_all.rb:93]: Yes, sure, good idea 👍

File: activerecord/lib/active_record/insert_all.rb [hunk 2]
```diff
@@ -10,15 +10,20 @@ class InsertAll # :nodoc:
...
+      @model, @connection, @inserts = model, model.connection, inserts
       @on_duplicate, @update_only, @returning, @unique_by = on_duplicate, update_only, returning, unique_by
       @record_timestamps = record_timestamps.nil? ? model.record_timestamps : record_timestamps
 
       disallow_raw_sql!(on_duplicate)
       disallow_raw_sql!(returning)
 
+      if has_attribute_aliases?(@inserts.first)
+        resolve_attribute_aliases
+      end
```
Comment (byroot) [inline activerecord/lib/active_record/insert_all.rb:22]: I think I would have put this check inside `resolve_attribute_aliases`. It's better for encapsulation.

**Review by byroot — APPROVED** [review]
Just one minor thing and then it's good for me. Ping me when it's done and I'll happily merge.

**Comment by fatkodima** [comment]
@byroot Updated 😺

**Pull request merged**